swarm/src/handler.rs

```diff
@@ -212,6 +212,38 @@ pub enum ConnectionEvent<'a, IP: InboundUpgradeSend, OP: OutboundUpgradeSend, IO
     ListenUpgradeError(ListenUpgradeError<IOI, IP>),
 }
 
+impl<'a, IP: InboundUpgradeSend, OP: OutboundUpgradeSend, IOI, OOI>
```

Comment:
Note that this needs a `libp2p-swarm` version bump and a bump of the `libp2p-swarm` dependency in the `libp2p-gossipsub` `Cargo.toml`.

Reply:
Right, because we released since we started this :)

Reply:
Done in bbdf8f5fb.